### PR TITLE
Add more file icon support for different audio files.

### PR DESCRIPTION
**Pull request by scottliubox**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @scottliubox has signed the CLA. Thanks for the pull request!

File: BoxBrowseSDK/BoxBrowseSDK/Helpers/BOXBrowseSDKFileTypeHelper.m [hunk 1]
```diff
@@ -15,7 +15,7 @@ @implementation BOXBrowseSDKFileTypeHelper
     static NSSet *extensions = nil;
     
     if (extensions == nil) {
-        extensions = [NSSet setWithObjects:@"aac", @"aiff", @"m3u", @"m4a", @"mid", @"mp3", @"wav", @"wpl", @"wma", @"amr", @"3gp", nil];
+        extensions = [NSSet setWithObjects:@"aac", @"aiff", @"m3u", @"m4a", @"mid", @"mp3", @"wav", @"wpl", @"wma", @"amr", @"3gp", @"alac", @"flac", @"ogg", @"m4r", nil];
```
Comment (boxtcarpel) [inline BoxBrowseSDK/BoxBrowseSDK/Helpers/BOXBrowseSDKFileTypeHelper.m]: maybe exclude ogg from this for now

Comment (boxtcarpel) [reply BoxBrowseSDK/BoxBrowseSDK/Helpers/BOXBrowseSDKFileTypeHelper.m]: public static function video_formats()
    {
        return ['3g2', '3gp', 'avi', 'm2v', 'm2ts', 'm4v', 'mkv', 'mov', 'mp4', 'mpeg', 'mpg', 'ogg', 'mts', 'qt', 'wmv'];
    }
    /**
     * @return array
     */
    public static function mobile_video_formats()
    {
        return ['flv'];
    }
    /**
     * @return array
     */
    public static function audio_formats()
    {
        return ['aac', 'aif', 'aifc', 'aiff', 'amr', 'au', 'flac', 'm4a', 'mp3', 'ra', 'wav', 'wma'];
    }

Comment (boxtcarpel) [reply BoxBrowseSDK/BoxBrowseSDK/Helpers/BOXBrowseSDKFileTypeHelper.m]: ^^ From Representations.php of the web app

File: BoxBrowseSDK/BoxBrowseSDK/Helpers/BOXBrowseSDKFileTypeHelper.m [hunk 2]
```diff
@@ -15,7 +15,7 @@ @implementation BOXBrowseSDKFileTypeHelper
     static NSSet *extensions = nil;
     
     if (extensions == nil) {
-        extensions = [NSSet setWithObjects:@"aac", @"aiff", @"m3u", @"m4a", @"mid", @"mp3", @"wav", @"wpl", @"wma", @"amr", @"3gp", nil];
+        extensions = [NSSet setWithObjects:@"aac", @"aiff", @"m3u", @"m4a", @"mid", @"mp3", @"wav", @"wpl", @"wma", @"amr", @"3gp", @"alac", @"flac", @"m4r", nil];
```
Comment (boxtcarpel) [inline BoxBrowseSDK/BoxBrowseSDK/Helpers/BOXBrowseSDKFileTypeHelper.m]: From Representations.php: 'aif', 'aifc', 'au', 'ra'

**Pull request merged**